12_quality_control.Rmd

```diff
@@ -293,6 +293,16 @@ plotColData(tse,"sum","SampleType", colour_by = "SampleType") +
     theme(axis.text.x = element_text(angle = 45, hjust=1))
 ```
 
+In addition, data can be rarefied with [subsampleCounts](https://microbiome.github.io/mia/reference/subsampleCounts.html), which normalises the samples to an equal number of reads. However, this practice is discouraged for the analysis of differentially abundant microorganisms (see [McMurdie PJ, Holmes S](https://doi.org/10.1371/journal.pcbi.1003531)).
+  
+### Contaminant sequences
+
+Samples might be contaminated with exogenous sequences. The impact of each contaminant can be estimated based on their frequencies and concentrations across the samples.
+
+The following [decontam functions](https://microbiome.github.io/mia/reference/isContaminant.html) support such functionality:
```

Comment:
Add citation to the literature reference on decontam pacakge: 
https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0605-2